### PR TITLE
Create Recovery Services Vault

### PR DESCRIPTION
**Pull request by nilaydshah**
This template creates a Recovery Services Vault which will be used further for Backup and SiteRecovery

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@nilaydshah__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (nilsha). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by nilaydshah** [comment]
@singhkay can you please review this PR and #1530 ?

File: 101-recoveryservices-vault-create/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Create Recovery Services Vault
```
Comment (singhkay) [inline 101-recoveryservices-vault-create/README.md]: @nilaydshah Let's call this folder 101-recovery-services-vault-create for readability

**Comment by nilaydshah** [comment]
@singhkay  Updated PR with folder name change

**Comment by singhkay** [comment]
lgtm, merged

**Pull request merged**